examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+export PORT_PROXY="${KAFKA_PORT_PROXY:-11100}"
+export PORT_ADMIN="${KAFKA_PORT_ADMIN:-11101}"
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+kafka_client () {
+    docker-compose run kafka-client "$@"
+}
+
+TOPIC="envoy-kafka-broker"
+
+MESSAGE="Welcome to Envoy and Kafka Broker filter!"
+
+run_log "Create a Kafka topic"
+kafka_client kafka-topics --bootstrap-server proxy:10000 --create --topic $TOPIC quickstart-events
```

Comment:
I might be out of date with this, but shouldn't it look like?
```
kafka_client kafka-topics --bootstrap-server proxy:10000 --create --topic $TOPIC --partitions 13 --replication-factor 1
```

Reply:
for the `--replication-factor 1`, we have `KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1` in docker-compose instead.